### DISCUSSION TITLE
Questions regarding mobile support

### DISCUSSION THREAD

**Discussion by ThatCodingKMC** [coder/code-server#7037] [Q&A] [answered]
I am using code-server through pwn.college which is a website to practice and gain cyber security related knowledge. While using the pwn.college VSCode workspace on mobile (through the browser) I cannot copy text out of the terminal. I believe the issue is not on pwn.college's end and instead due to something related to how code-server's display interacts with mobile devices. I can't even highlight the text in the terminal. Any suggestions on if I am doing something wrong or potential bug fixes? I am on Android in case that matters. Thanks!

For the record I am not a dev of pwn.college and don't fully understand how they have things setup, but it appears to me that the issue is only on mobile and only for the terminal. On mobile I am able to copy other stuff in the VSCode workspace, just nothing in the terminal.

**Comment by code-asher** [comment]
We are just wrapping VS Code web, so the mobile support question might get better answers upstream at https://github.com/microsoft/vscode.

I am not seeing these particular issues though, so maybe they are unknown: https://github.com/microsoft/vscode/issues?q=is%3Aissue%20state%3Aopen%20label%3Aandroid%20

**Reply by code-asher** [reply]
One thing to test is if you get the same problem in Codespaces.  If you do, it is an upstream issue.  If not, it is either a problem with code-server or a problem with the website.

**Reply by ThatCodingKMC** [reply, marked as answer]
Looks like the issue is upstream, I was able to recreate it in Codespaces. Thanks for helping out!
